README.md

```diff
@@ -9,6 +9,19 @@ This is a tool to help with managing of resources in [IBM Power Systems Virtual
 2. Select the latest release and download the relevant binary under the Assets section.
 3. Run the `pvsadm --help` command to check the available subcommands and the options.
 
+### OSX Installation
```

Comment:
lets create previous section from release page as another section so that aligns well with `OSX installation`

Reply:
btw: This is not just osx, I see we are shipping for intel linux as well, lets make if generic like Homebew section and in a note you can mention we are shipping for only Mac(x86&arm) and Linux(x86)

Reply:
Done. Added a new section `Binary` for installation using binary  and mentioned the supported architecture for brew installation.